beetmoverscript/utils.py

```diff
@@ -15,21 +15,44 @@ def load_json(path):
         return json.load(fh)
 
 
+def infer_template_args(context):
+    props = context.properties
+    tmpl_key_option = "signed" if context.task["payload"]["update_manifest"] is True else "unsigned"
```

Comment:
Yep, you're right. I'll address this now here but I'll add a separte issue on this repo to document everything. I left docstrings, travis, tests, coverage and others for when we have a stable working version.